README.md

```diff
@@ -17,6 +17,104 @@ You can install `SymEngine.jl` by giving the following command.
 julia> Pkg.add("SymEngine")
 ```
 
+## Quick Start
+
+### Working with scalar variables
+
+#### Defining variables 
+One can define variables in a few ways the following 3 examples are equivent.
+Defining 2 variables `a`  and `b` with the names `a` and `b`.
+
+``` julia
+julia> a=symbols(:a); b=symbols(:b)
+b
+
+julia> a,b = symbols("a b")
+(a, b)
+
+julia> @vars a b
+(a, b)
+```
+
+`@vars` is perhaps the most handy for basic use, as it creates a varible whos julia name and whos symbolic name match.
+Which is what one normally wants.
+
+### Simple expressions
+We are going to define an expression using the variables from earlier:
+
+``` julia
+julia> ex1 = a + 2(b+2)^2 + 2a + 3(a+1)
+3*a + 3*(1 + a) + 2*(2 + b)^2
+```
+
+One can see that values are grouped, but no expansion is done.
+
+
+### Working with vector and matrix variables
+
```

Comment:
You might want to mention that these just use overloads on standard Julia functions and constructs, so no specialized package knowledge is needed.